ground/src/main/java/com/google/android/ground/Config.kt

```diff
@@ -55,10 +55,11 @@ object Config {
   const val MAX_MEDIA_UPLOAD_RETRY_COUNT = 5
 
   // TODO(#1730): Make sub-paths configurable and stop hardcoding here.
+  const val DEFAULT_MOG_TILE_URL = "/offline-imagery/default/"
```

Comment:
Iirc we may be adding the trailing `/` twice - let's remove it here? Also, not this is no longer a URL - Can we rename to `DEFAULT_MOG_TILE_LOCATION`?